pkl-core/src/main/java/org/pkl/core/runtime/VmTyped.java

```diff
@@ -130,6 +131,13 @@ public VmMap getClassMirrors() {
     return builder.build();
   }
 
+  public @Nullable VmClass getClass(Identifier name) {
+    assert isModuleObject();
+    if (getModuleInfo().isAmend()) return null;
+    return (VmClass)
+        getMember(it -> it.isClass() && it.getName() == name); // Note: includes local classes
+  }
+
```

Comment:
This and `getTypeAlias(Identifier name)` are unused, remove

Note: the correct way to do this is to use `VmUtils.readMember`

Reply:
Ah, good catch. These were left over from the iteration that included the in-language decoder.